x/accounts/go.mod

```diff
@@ -40,8 +40,9 @@ require (
 	cosmossdk.io/log v1.5.0 // indirect
 	cosmossdk.io/math v1.5.0
 	cosmossdk.io/schema v1.0.0
-	cosmossdk.io/store v1.10.0-rc.1 // indirect
-	cosmossdk.io/x/staking v0.0.0-00010101000000-000000000000 // indirect
+	cosmossdk.io/store v1.10.0-rc.1.0.20241218084712-ca559989da43 // indirect
+	cosmossdk.io/x/accounts/defaults/lockup v0.2.0-rc.1
```

Comment:
any chance we can avoid this import?

Reply:
Msgs for lockup account were added in v0.2.0. 
I think this code is needed to import the type from cosmossdk.io.

Reply:
If you can register the type in another way, you may replace it.

Reply:
We can register them in the app.go directly maybe.  Could be something we add to the x/accounts documentation.

Reply:
If adding to x/accounts, this change would be a good way to do it. 
Changing x/accounts.go.mod has less impact than changing the entire go.mod.

Reply:
Simapp is its own go.mod, so it would have no impact to do the registration there and be only beneficial.

Reply:
Should the lockup type registration be done in each chain's app.go?
I think it should be completed in depinject in x/accounts.

Reply:
I have reverted  it once.
https://github.com/cosmos/cosmos-sdk/issues/23312
Do you know anything about the problem of there being no query to retrieve lockup accounts owned by an address in x/accounts?